app/dataBuilder/generator/generate.py

```diff
@@ -80,6 +81,36 @@ def random_number_string(amount, minimum=0, maximum=9):
     return "".join([str(random.randint(minimum, maximum)) for _ in range(amount)])
 
 
+def return_random_color(file_name):
+    with open(absolute_path(file_name), encoding="utf8") as f:
+        colors = [line for line in f]
+        color = random.choice(colors)
+    return color.rstrip('\n')
+
+
+def generate_license_plate():
+    chars_list = [random.choice(string.ascii_uppercase) for _ in range(3)]
+    nums_list = [random.choice(range(1,9)) for _ in range(3)]
+    license_plate = ''.join(map(str, chars_list)) + ' ' + ''.join(map(str, nums_list))
+    return license_plate
+
+
+def generate_car():
+    with open('files/carbrands_models.txt', encoding="utf8") as f:
```

Comment:
absolute_path('carbrands_models.txt')
instead of 'files/carbrands_models.txt'
for more flexibility